src/Workload/Shared/FrameworkList.targets

```diff
@@ -3,8 +3,6 @@
   <ItemGroup>
     <_FrameworkListFile Condition=" !$(MSBuildProjectName.Contains('.Runtime')) " Include="$(IntermediateOutputPath)FrameworkList.xml" />
     <_FrameworkListFile Condition=" !$(MSBuildProjectName.Contains('.Ref')) " Include="$(IntermediateOutputPath)RuntimeList.xml" />
-    <PackageReference Include="Microsoft.DotNet.Build.Tasks.Packaging"  Version="6.0.0-beta.21169.1" PrivateAssets="all" />
```

Comment:
This seems unrelated. Can it be a separate, independent PR?

Reply:
Yea this was needed so I could build the workload
I rebased now that https://github.com/dotnet/maui/pull/6130 merged